lib/options.js

```diff
@@ -35,13 +44,22 @@ function loadOptions(fileName, key, options) {
         return false;
       }
     }
+    if (overrides.extends) {
+      var shareableConfigPath = getShareAbleConfigPath(overrides);
+      if (!loadOptions(path.resolve(shareableConfigPath, '.min-wd'), null,
+        options)) {
+        overrides = loadOptions(path.resolve(shareableConfigPath,
+          'package.json'), 'webdriver', options);
+      }
+    }
+
     override(options, overrides);
     return true;
   }
   return false;
 }
 
-module.exports = function (opts) {
+module.exports.retrieve = function retrieve(opts) {
```

Comment:
`exports` is globally available within the file. You can omit `module`.

Reply:
Of course :-)

Reply:
Have a look. I'm using `process.cwd()` as the base directory to allow for testing.

Reply:
Fixed